src/main/java/build/buildfarm/instance/AbstractServerInstance.java

```diff
@@ -466,6 +474,14 @@ public String listOperations(
   }
 
   @Override
+  public void deleteOperation(String name) {
```

Comment:
This method doesn't actually do anything yet. What's the plan here?

Reply:
Sorry, not sure if my update was clear, it does do something - the remove is within the if block (probably shouldn't have side effects there with the test).  I'll update